README_Translations/README_id_ID.md

```diff
@@ -0,0 +1,85 @@
+<h1 align="center">
+  <br>
+  <a href="http://atlasos.net"><img src="https://i.imgur.com/xV08gIt.png" alt="Atlas" width="900"></a>
+</h1>
+<h4 align="center">Sistem operasi Windows yang open source dan transparan, dirancang untuk mengoptimalkan.</h4>
+
+<p align="center">
+  <a href="https://github.com/Atlas-OS/Atlas/wiki/2.-Installing">Instalasi</a>
+  •
+  <a href="https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#contents">FAQ</a>
+  •
+  <a href="https://discord.com/servers/atlas-795710270000332800" target="_blank">Discord</a>
+</p>
+<p align="center">
+ Bahasa Lainnya:
+  <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_zh_CN.md">简体中文</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_fr_FR.md">Français</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_pl_PL.md">Polski</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_ru_RU.md">Pусский</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_id_ID.md">Bahasa Indonesia</a>
```

Comment:
English should be first in the translations. and the language you are translating shouldn’t be in the list as would be a loop

Reply:
fix at https://github.com/Atlas-OS/Atlas/pull/454/commits/187aad7a9e489d3603f7f8959deca37787805a98